### DISCUSSION TITLE
Options in oxalis.conf

### DISCUSSION THREAD

**Discussion by ndTEC** [OxalisCommunity/oxalis#584] [Q&A]
Hi,
Is there anywhere documentation about every possible option/parameter that can be configured in oxalis.conf?

**Comment by RichardVanMaaren** [comment]
Good questions, I am looking for basically the same. For example in the oxalis-standalone you have the evidencePath parameter (-e)  Is something like this also available for the oxalis.war that for example for incoming we can set the evidencePath via the oxalis.conf ? I checked the code of oxalis and AS4 but could not find this. Anybody suggestions ? Or can answer above questions ? ;-)

Best regards,

Richard

**Comment by RichardVanMaaren** [comment]
> Hi, Is there anywhere documentation about every possible option/parameter that can be configured in oxalis.conf?

Hi,

Anything specific you are looking for ? I think there is no real document for this but I can tell you where to find it.

In oxalis-master\oxalis-commons\src\main\java\network\oxalis\commons for example persist you have a file called PersisterConf.java which hold entries for the oxalis.conf. I checked the other directories and they have a similar file in there. Hope this helps a little bit.

In general a good idea to create a document and document how these entries can be set.

Best regards,

Richard

**Comment by aaron-kumar** [comment]
Unfortunately not all oxalis.conf configurations are documented. But you can refer _all_  configurations by searching reference.conf (or *Conf.Java) for those properties (_though linked with plugin that you use_)

@ndTEC  and @RvanMaarenApro please feel free to document these setting. Probably right place for this documentation is : https://github.com/OxalisCommunity/oxalis/wiki . Please use that..
